.ci/run.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import stat
+import platform
+import subprocess
+import tempfile
+import logging
+from contextlib import contextmanager
+from tabulate import tabulate
+import colorama
+
+
+FAIL_FAST = os.getenv("FAIL_FAST", "0").lower() in ["1", "y", "yes", "true"]
+LOGGING_LEVEL = int(os.getenv("CONAN_LOGGING_LEVEL", logging.INFO))
+logging.basicConfig(format='%(asctime)s [%(levelname)s]: %(message)s', level=LOGGING_LEVEL)
+
+
+@contextmanager
+def chdir(dir_path):
+    current = os.getcwd()
+    os.chdir(dir_path)
+    logging.debug("cd {}".format(dir_path))
+    try:
+        yield
+    finally:
+        logging.debug("cd {}".format(current))
+        os.chdir(current)
+
+
+def writeln_console(message):
+    sys.stdout.write(message)
+    sys.stdout.write('\n')
+    sys.stderr.flush()
+    sys.stdout.flush()
+
+
+def get_build_list():
+    builds = []
+    script = "build.bat" if platform.system() == "Windows" else "build.sh"
+    for root, dirs, files in os.walk("."):
+        if "build.py" in [os.path.basename(file) for file in files]:
+            builds.append(os.path.join(root, "build.py"))
+            continue
+
+        for file in files:
+            if os.path.basename(file) == script:
```

Comment:
```suggestion
            if os.path.basename(file) in [script, 'build.py']:
                builds.append(os.path.join(root, file))
                break  # End the for loop
```

and you can remove the previous `if` block

Reply:
done